crates/ruff_benchmark/benches/linter.rs

```diff
@@ -79,7 +79,7 @@ fn benchmark_default_rules(criterion: &mut Criterion) {
 
 fn benchmark_all_rules(criterion: &mut Criterion) {
     let settings = Settings {
-        rules: RuleSelector::All.into_iter().collect(),
+        rules: RuleSelector::All.all_rules().collect(),
```

Comment:
Presumably the benchmark regressions are due to including our nursery/preview rules now

Reply:
Yep https://github.com/astral-sh/ruff/pull/7195/commits/9522dfbc0119727b3c8ec8a7b0991159c6229c8c — I'll do this in a separate PR to avoid blocking this one

Reply:
#7208